apps/real-time-collaboration/README.md

```diff
@@ -0,0 +1,75 @@
+
+## Prerequisites
+
+Before getting started, you must have the following prerequisites:
+
+* An [AWS account](https://signin.aws.amazon.com/signin?redirect_uri=https%3A%2F%2Fportal.aws.amazon.com%2Fbilling%2Fsignup%2Fresume&client_id=signup)
+that provides access to AWS services.
+* Ensure that you have installed [Node.js](https://nodejs.org/en/) version 14 or higher.
+
+## Getting started
+
+
+### Creating AWS resources
+
+1. Sign in to the AWS Management Console with your primary account. Switch to the *us-east-1 (N. Virginia)* Region. 
+   Note: The AWS CloudFormation (https://aws.amazon.com/cloudformation/) template in this section needs to be 
+   launched in US east (N. Virginia) Region.
+2. Copy the contents of the AWS CloudFormation template from real-time collaboration sample app in
+   [Github](https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/real-time-collaboration/backend/template.yaml) and 
+   save it in a new file named real-time-collaboration-app-template.yaml.
+3. Open the AWS CloudFormation console (https://console.aws.amazon.com/cloudformation/) and choose *Create stack*.
+4. Choose **Upload a template file**, and then browse for *template.yaml* in the *./backend* folder.
+5. For **Stack name**, type real-time-collaboration-app-demo and choose **Next**.
+6. On the **Specify Details** page, enter the stack name: **DemoName-ChimeCollabDemo**
+7. Choose **Next**, and then **Next** on the **Configure stack options** page.
+8. On the **Review** page, check the **I acknowledge that AWS CloudFormation might create IAM resources** check box. Then click **Create**.
+9. Creating the stack generates 3 outputs: **ApiGatewayUrl, AppInstanceArn, AdminUserArn**. Note these values 
+   for the outputs to use to configure the app in the next step.
+
+### Install dependencies and build the packages. 
+
+```
+npm install
+npm run build
+```
+
+### Running the text editor demo application locally
+
+1. Run the following commands to navigate to the `text-editor` folder:
+
+```shell
+cd ./amazon-chime-sdk/apps/real-time-collaboration/examples/text-editor
+```
+
+2. Open src/Config.js with the editor of your choice. Add the following configuration to it:
+
+```js
+const appConfig = {
+ApiGatewayUrl: '<API Gateway URL from CloudFormation Outputs>',
+AppInstanceArn: '<appInstanceArn from CloudFormation Outputs>',
+AdminUserArn: '<adminUserArn from CloudFormation Outputs>'
+};
+export default appConfig;
+```
+
+3. Once the configuration for the application is entered, run the following commands in the text editor folder.
+
+```
+npm run start
+```
+
+4. Open your browser and navigate to https://localhost:8080 (https://localhost:8080/) to start testing. 
+   The application updates the url with the Amazon Chime channel created for this session. 
+   Type a few words in the text editor.
+6. Copy the url with the Amazon Chime channel into a new web browser window. The new browser window shows 
```

Comment:
Step 5 is missing. In the markdown, you can use `1` without a blank link for numbering:

```
1. Run the following commands... 
1. Open src/Config.js with ...
1. Once the configuration for...
```

This will show up as 1, 2, 3...